docs/conf.py

```diff
@@ -253,6 +253,3 @@ def rstjinja(app, docname, source):
 def setup(app):
     # Generate the stability page
     app.connect("source-read", rstjinja)
-
-    # The theme conf provides a fix for circle ci redirections
-    fix_circleci(app)
```

Comment:
Was this meant to be deleted?

Reply:
yeah, we don't need to do this any more as we aren't previewing on circle.

Reply:
Oh yeah!